### PR TITLE
Update/rules

### LINKED ISSUE #24: Update Rules Text
Once the post battle screen has been completed. We will need to update our rules. Our rules have changed since the first iteration and are a little outdated.

- [x] Update rule text to be accurate with how the game is now played.

### PR DESCRIPTION
**Pull request by timjacksonm**
Just updated the text for rules. Prettier made a few adjustments of its own also.l

### CODE REVIEW

File: src/components/Rules/index.js [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
-import { IoMdCloseCircle } from "react-icons/io";
+import { IoMdCloseCircle } from 'react-icons/io';
 
-const Rules = ({setShown}) => {
-
-  return(
+const Rules = ({ setShown }) => {
+  return (
     <Rule id="modal">
       <h1>Dino-Battle!</h1>
```
Comment (savwiley) [inline src/components/Rules/index.js]: This one was my fault for overlooking it, but we should probably change the title to "Dino-War" instead to match with the home screen and tab. Or perhaps change it to just "Rules", "How-To", or something.

File: src/components/Rules/index.js [hunk 2]
```diff
@@ -1,61 +1,74 @@
...
+        </p>
+        <p>
+          Be careful! The game will add in subtraction dinosaurs as the
+          difficulty increases.
+        </p>
+        <p>
+          The player will adjust the input field to equal each teams total point
+          value.
+        </p>
+        <p>Once you are certain of your answer click Battle!</p>
```
Comment (savwiley) [inline src/components/Rules/index.js]: Needs a comma here. 

`...your answer, click Battle!`

File: src/components/Rules/index.js [hunk 3]
```diff
@@ -1,61 +1,74 @@
...
+        <h2>How is the score calculated?</h2>
+        <p>
+          Once the user starts the battle, the game will count each dinosaurs
```
Comment (savwiley) [inline src/components/Rules/index.js]: Apostrophe needed.

`...the game will count each dinosaur's...`

File: src/components/Rules/index.js [hunk 4]
```diff
@@ -1,61 +1,74 @@
...
+          value starting with the red team.
+        </p>
+        <p>
+          The computer will place each teams total value in the actual count
```
Comment (savwiley) [inline src/components/Rules/index.js]: Apostrophe needed.

`The computer will place each team's total...`

File: src/components/Rules/index.js [hunk 5]
```diff
@@ -1,61 +1,74 @@
...
+          container.
+        </p>
+        <p>
+          If atleast one of your answers is equal to the actual count value, you
```
Comment (savwiley) [inline src/components/Rules/index.js]: Typo.

`If at least one...`

**Review by savwiley — COMMENTED** [review]
My review is just syntax/grammar stuff. The one coding related thing I'm curious about are the new line breaks that Prettier likely added. I just want to make sure the lines aren't breaking weird in the browser. (I was going to download GH CLI to check it myself but it was taking awhile, so I'll leave it to your judgement.)

But since the code is otherwise fine, I won't officially request changes. It's okay to merge.

File: src/components/Rules/index.js (hunk 1 above)
Comment (timjacksonm) [reply src/components/Rules/index.js]: Changed to Dino-War. I think its okay to have the title of the game at the top.

**Pull request merged**